ReactAndroid/src/main/java/com/facebook/react/modules/network/OkHttpClientProvider.java

```diff
@@ -38,12 +38,23 @@ public static void replaceOkHttpClient(OkHttpClient client) {
   }
 
   private static OkHttpClient createClient() {
+
+    // create a list of protocols the client will use to communicate
+    // by default, it will use the most efficient one, falling back to 
+    // more ubiquitous ones
+    // above as per okhttp javadocs
+    List<Protocol> protocolList = new ArrayList<>();
+    protocolList.add(Protocol.HTTP_2);
+    protocolList.add(Protocol.SPDY_3);
+    protocolList.add(Protocol.HTTP_1_1);
```

Comment:
The PR that you referenced referenced here - https://github.com/wallabag/android-app/pull/256/files

It works around the issue by removing Http2 support.